src/system/platform/component/app/GUI/Component.ts

```diff
@@ -204,20 +204,20 @@ export default class GUI extends Element<HTMLDivElement, Props> {
     )
     this._color_picker = color_picker
 
-    const color_pallete = new Div(
+    const color_palette = new Div(
       {
         style: {
-          className: 'gui-color-pallete',
+          className: 'gui-color-palette',
```

Comment:
I didn't see anything referencing this class name that would've needed updating.